mlir/test/Target/LLVMIR/openmp-parallel-do-simd.mlir

```diff
@@ -0,0 +1,30 @@
+// RUN: mlir-translate -mlir-to-llvmir %s | FileCheck %s
+
+// Check that omp.simd as a leaf of a composite construct still generates
+// the appropriate loop vectorization attribute.
+
+// CHECK-LABEL: define internal void @test_parallel_do_simd..omp_par
+// CHECK: ![[VAL:.*]] = !{!"llvm.loop.vectorize.enable", i1 true}
```

Comment:
Minimal tests are good but these are a bit too minimal. Could you test enough to show each construct and which loop is being vectorized.